src/control/lib/spdk/ctests/SConscript

```diff
@@ -0,0 +1,23 @@
+"""Build spdk bindings C tests"""
+import daos_build
+
+def scons():
+    """Execute build"""
+    Import('senv', 'prereqs')
+
+    unit_env = senv.Clone()
+    #unit_env.AppendUnique(OBJPREFIX='utest_')
+    # Some syscalls are aliased when fortification is enabled, which makes
+    # mocking in unit tests into a headache.
+    # The compiler decides this on its own, but let's force the issue.
+    #unit_env.Append(CCFLAGS="-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0")
+
+    prereqs.require(unit_env, 'pmdk', 'spdk', 'hwloc', 'cmocka')
+
+    daos_build.test(unit_env, 'nvme_discover_tests',
+                    ['nvme_discover_test.c', senv.ncc, senv.nc],
```

Comment:
(pylint-no-member) Instance of 'DefaultEnvironment' has no 'ncc' member
(pylint-no-member) Instance of 'DefaultEnvironment' has no 'nc' member